web/src/InstallationProgress.jsx

```diff
@@ -49,6 +50,9 @@ function InstallationProgress() {
 
   // FIXME: this is an example. Update or drop it.
   const Messages = () => {
+    if (status !== 3)
```

Comment:
Now that we merge #65, you could use the constants we defined for statuses.

Reply:
Right!